deploy/hubble/manifests/controller/helm/retina/templates/agent/daemonset.yaml

```diff
@@ -196,8 +208,8 @@ spec:
             - .\setkubeconfigpath.ps1; ./controller.exe --config ./retina/config.yaml --kubeconfig ./kubeconfig
           readinessProbe:
             httpGet:
-              path: /metrics
-              port: {{ .Values.agent.container.retina.ports.containerPort }}
+              path: /healthz
+              port: {{ .Values.agent.container.retina.ports.healthPort }}
```

Comment:
@matmerr Question on the windows configurations in the hubble control plane manifests. Is there a reason we have this or should we do some housekeeping?